tests/test_api.py

```diff
@@ -720,38 +740,43 @@ async def test_row_foreign_key_tables(ds_client):
     assert response.json()["foreign_key_tables"] == [
         {
             "other_table": "foreign_key_references",
-            "column": "id",
-            "other_column": "foreign_key_with_blank_label",
+            "columns": ["id"],
+            "other_columns": ["foreign_key_with_blank_label"],
             "count": 0,
             "link": "/fixtures/foreign_key_references?foreign_key_with_blank_label=1",
+            "other_columns_reference": "foreign_key_with_blank_label",
```

Comment:
this is also another API change. this got introduced so we could have compound keys look good in the row template, but we could do that through a custom filter instead.